### PR TITLE
 Arcconf Plugin: Add 'volume-ident-led-off' support

### PR DESCRIPTION
**Pull request by sharath-t**
implementation for 'volume-ident-led-off' API support

### CODE REVIEW

File: plugin/arcconf/arcconf.py [hunk 1]
```diff
@@ -644,3 +644,187 @@ def volume_delete(self, volume, flags=0):
...
+            raise LsmError(
+                ErrorNumber.INVALID_ARGUMENT,
+                "Illegal input volume argument: missing plugin_data property")
+
+        volume_info = volume.plugin_data
+        ctrl_id = str(volume_info['ctrl_id'])
+        volume_id = str(volume_info['ld_id'])
+
+        try:
+            self._arcconf_exec(['IDENTIFY', ctrl_id, 'LOGICALDRIVE', volume_id, 'STOP'], flag_force=True)
```
Comment (cathay4t) [inline plugin/arcconf/arcconf.py]: The text width should be less than 80.

File: plugin/arcconf/arcconf.py [hunk 2]
```diff
@@ -644,3 +644,187 @@ def volume_delete(self, volume, flags=0):
...
+        except ExecError:
+            raise LsmError(ErrorNumber.PLUGIN_BUG, 'Volume-ident-led-off failed unexpectedly')
```
Comment (cathay4t) [inline plugin/arcconf/arcconf.py]: The text width should be less than 80.

File: plugin/arcconf/arcconf.py (hunk 1 above)
Comment (sharath-t) [reply plugin/arcconf/arcconf.py]: done - commit id - feb56754a6df51c945c854845a18460800378573

File: plugin/arcconf/arcconf.py (hunk 2 above)
Comment (sharath-t) [reply plugin/arcconf/arcconf.py]: done - commit id - feb56754a6df51c945c854845a18460800378573

**Comment by tasleson** [comment]
Integrated into https://github.com/libstorage/libstoragemgmt/pull/386

**Pull request closed without merging**